spec/volcanic/authenticator/v1/identity_spec.rb

```diff
@@ -206,4 +206,12 @@
       it { expect(identities[2].id).to eq 3 }
     end
   end
+
+  describe '#deactivate!' do
```

Comment:
It would be good to see some more tests.

What happens when:
* the identity doesn't exist?
* the identity does exist and is currently active?
* the requester doesn't have permission to deactivate that identity?

Reply:
done